dcpy/builds/load.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+from dcpy.utils import postgres
+from dcpy.connectors.edm import recipes
+
+
+def load_source_data(
+    pg_client: postgres.PostgresClient,
+    recipe_path: Path,
+):
+    recipe_lock_path = recipe_path.parent / "recipe.lock.yml"
+    recipes.plan(
+        recipe_path,
+        recipe_lock_path,
+    )
+    recipe = recipes.recipe_from_yaml(Path(recipe_lock_path))
+    recipes.write_source_data_versions(recipe_file=Path(recipe_lock_path))
+
+    [recipes.import_dataset(dataset, pg_client) for dataset in recipe.inputs.datasets]
+
+    pg_client.create_table_from_csv(
```

Comment:
Do we still need the pg table for source data versions?

Reply:
it's been nice to be able to see & query the versions

Reply:
huh... do we need to query the versions in the db? ie. just curious if the csv isn't enough for some reason

Reply:
being able to query the resolved recipe versions is probably useful and yea seeing the info in dbeaver when a build starts is really nice. it's kind of like a seed file/table